observIQ/bindplane-agent
# feat: Synthetic Telemetry Generator Receiver

### Proposed Change

Adds a receiver that produces synthetic telemetry based on the config. Currently only supports logs.

### Testing

Configure the agent with the following custom logs source:
```yaml
telemetrygeneratorreceiver:
    payloads_per_second: 1
    generators:
        - type: logs
          resource_attributes:
              res_key1: res_value1
              res_key2: res_value2            
          attributes:
              attr_key1: attr_value1
              attr_key2: attr_value2            
          additional_config:
              body: this is the body   
              severity: 4
        - type: logs
          resource_attributes:
              res_key3: res_value3
              res_key4: res_value4            
          attributes:
              attr_key3: attr_value3
              attr_key4: attr_value4            
          additional_config:
              body: this is another body   
              severity: 10         
```

Send it to any destination (custom `nop:` is fine), snapshot console should show logs flowing.

##### Checklist
